### ISSUE TITLE
Install basic counter for traffic

### ISSUE THREAD

**Issue by bebopkid** [QiHardware/scale.qihardware.org#8]
Will be nice to see traffic visiting the Scale site and the number of PDF downloads per issue

**Comment by clemsos** [comment]
Could install https://github.com/matomo-org/matomo on some server and use it for that

**Comment by rejon** [comment]
Yes, better. Remove 3rd party services where possible from the big companies where crypto involved.

**Comment by clemsos** [comment]
yah stay away for google analytics for sure. 

The easiest way for now is maybe to use bitly to link the PDF so keep tracks of downloads. Anyone has tried that before ?

**Comment by rejon** [comment]
back into the frying pan haha, have used, but I use people and conversations as my metric, followup, users in telegram, real tweets, etc
